flathub.yaml

```diff
@@ -0,0 +1,2 @@
+only-arches:
+  - x86_64
```

Comment:
this file is ignored because it is flathub.json.

But the aarch64 build passed, so why making it x86_64?